NeuroID/NeuroIDClass/Extensions/NIDPacketNumber.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  NIDPacketNumber.swift
+//  NeuroID
+//
+//  Created by jeff on 7/15/24.
+//
+
+import Foundation
+
+public extension NeuroID {
+    static func getPacketNumber() -> Int32 {
+        return packetNumber
+    }
+    
+    static func incrementPacketNumber () {
+        packetNumber += 1
+    }
```

Comment:
Are we forever incrementing? Does this number need to be reset to 0 on a new session or?

Reply:
Oooh good call!! I’ll make a reset

Reply:
Added reset when configure is called... this should closely match what happens in web